### PR TITLE
salesforce faculty verification

### PR DESCRIPTION
**Pull request by rich-hart**
This is just the basic functionality of connecting to salesforce.  Several features still need to be implemented, .e.g. updating the authentication method to use OAuth 2.0, tracking salesforce usage statistics, ~~encrypted keys for travis testing,~~ etc.

### CODE REVIEW

**Comment by rich-hart** [comment]
This is just the basic functionality of connecting to salesforce.  Several features still need to be implemented, .e.g. updating the authentication method to use OAuth 2.0, tracking salesforce usage statistics, etc.

File: accounts/salesforce.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from simple_salesforce import Salesforce
+from django.conf import settings
+
+
+sf=Salesforce(username=settings.SALES_USER,
+              password=settings.SALES_PASSWORD,
+              security_token=settings.SALES_SECURITY_TOKEN)
+
+
+def faculty_status(accounts_id):
```
Comment (rich-hart) [inline accounts/salesforce.py]: Leaving a reminder to myself to add error handling here.

Comment (mwharrison) [inline accounts/salesforce.py]: Yeah - there was a question from Kathi about what would happen if SF was down - how will the app handle that? This should be addressed in the error handling.

File: accounts/salesforce.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    def __exit__(self, *exc):
+        if not exc==(None,None,None):
+            (exc_type, exc, exc_tb)=exc
+            error_type = exc_type.__name__
+            error_message = str(exc)
+            logger.error("{0}('{1}')".format(error_type,error_message))
+        return False
+
+    def faculty_status(self,accounts_id):
```
Comment (mwharrison) [inline accounts/salesforce.py]: Can you add spaces after the commas in this file to maintain readability?

Comment (rich-hart) [inline accounts/salesforce.py]: Sure thing.  I'll make a PEP8 update

**Comment by mwharrison** [comment]
I am getting the following error when I hit /api/user:

```
'Settings' object has no attribute 'SALESFORCE'
```

I see you added a template to local.py.example - how do I get the keys to test this?

**Comment by rich-hart** [comment]
I just sent you the keys using lastpass.

**Pull request merged**